webcompat/form.py

```diff
@@ -100,6 +114,15 @@ def get_problem(category):
     return u'Unknown'
 
 
+def get_browser_test(category):
```

Comment:
the name is probably not good too. Something like `get_tested_elsewhere()` and it should probably return True/False. but as said in other comments, this function has not been written. yet.

It probably would be good to add tests for most of the functions in this code. I realized we have never written tests for all the functions. And it's probably why we don't get to know if it's legit or not. 

Proposal: Add tests for the new functions you created. Testing input/output. And open a new issue for adding tests to all the functions you have not touched in this PR and assign it to me if you wish. 
https://github.com/webcompat/webcompat.com/blob/master/tests/test_form.py